### PR TITLE
Fix #360 unverified breakpoint

### LINKED ISSUE #360: Unverified breakpoint
_From @1dnmr on June 9, 2016 13:18_
- VSCode Version: 1.2.0
- OS Version: OS X Yosemite 10.10.5

Steps to Reproduce:
1. Install latest version of Visual Studio Code with all necessary extension for Go lang developing according to this article https://github.com/Microsoft/vscode-go
2. Set breakpoint and start debugging. 

Breakpoint become grey. If point cursor on them you will see "Unverified Breakpoint" hint appears.
I've searched how to fix it but nothing is helped.

_Copied from original issue: Microsoft/vscode#7475_

### PR DESCRIPTION
**Pull request by llife0915**

### CODE REVIEW

**Comment by ramya-rao-a** [comment]
@llife0915 Can you please explain what issue you are trying to fix and how?

Also, there are linting errors with your code. Run `npm run lint` and you should be able to see the errors in your machine

**Comment by ramya-rao-a** [comment]
Thanks @llife0915 
@roblourens Can you take a look?

File: src/debugAdapter/goDebug.ts [hunk 1]
```diff
@@ -511,25 +521,25 @@ class GoDebugSession extends DebugSession {
...
-			}));
-		}).then(newBreakpoints => {
-			verbose('All set:' + JSON.stringify(newBreakpoints));
-			let breakpoints = newBreakpoints.map((bp, i) => {
-				if (bp) {
-					return { verified: true, line: bp.line };
-				} else {
-					return { verified: false, line: args.lines[i] };
-				}
+			})).then(newBreakpoints => {
```
Comment (roblourens) [inline src/debugAdapter/goDebug.ts]: It looks like you moved this code up a level in the promise chain (see the indentation). Was that necessary? If not, can you leave it the same?

Comment (llife0915) [inline src/debugAdapter/goDebug.ts]: That was not necesary.  I leave it the same.

**Pull request merged**